.github/CONTRIBUTING.md

```diff
@@ -180,22 +195,29 @@ To do so, login to [CircleCI](https://app.circleci.com/) and enable your forked
 
 We welcome any useful contribution! For your convenience here's a recommended workflow:
 
-0. Think about what you want to do - fix a bug, repair docs, etc.
-1. Start your work locally (usually until you need our CI testing)
-   - create a branch and prepare your changes
-   - hint: do not work with your master directly, it may become complicated when you need to rebase
-   - hint: give your PR a good name! it will be useful later when you may work on multiple tasks/PRs
-2. Create a "Draft PR" which is clearly marked, to let us know you don't need feedback yet.
-3. When you feel ready for integrating your work, mark your PR "Ready for review".
-4. Use tags in PR name for following cases:
-   - **[blocked by #<number>]** if you work is depending on others changes
-   - **[wip]** when you start to re-edit your work, mark it so no one will accidentally merge it in meantime
+0. Think about what you want to do - fix a bug, repair docs, etc. If you want to implement a new feature or enhance an existing one, start by opening a GitHub issue to explain the feature and the motivation. Memebrs from core-contributors will take a look (it might take some time- we are often overloaded with issues!) and discuss it. Once an agreement was reached- start coding.
```

Comment:
```suggestion
0. Think about what you want to do - fix a bug, repair docs, etc. If you want to implement a new feature or enhance an existing one, start by opening a GitHub issue to explain the feature and the motivation. Members from core-contributors will take a look (it might take some time- we are often overloaded with issues!) and discuss it. Once an agreement was reached - start coding.
```